### PR TITLE
Rework attributes

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | #62 

⚠️ Merge after release of https://github.com/yiisoft/middleware-dispatcher/pull/68.

### CODE REVIEW

**Review by vjik — COMMENTED** [review]
Overall good! Need tests

File: README.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ composer require yiisoft/request-model
 
 ## General usage
 
+According to [`yiisoft/middleware-dispatcher`](https://github.com/yiisoft/middleware-dispatcher) docs, you need to tweak
```
Comment (samdark) [inline README.md:35]: Can't get what is mean here :(

**Review by samdark — APPROVED** [review]

File: src/HandlerParametersResolver.php [hunk 2]
```diff
@@ -42,19 +48,22 @@ private function getAttributeParams(array $parameters, ServerRequestInterface $r
...
-                    ),
-                    HandlerParameterAttributeInterface::QUERY_PARAM => $request
-                        ->getQueryParams()[$attributeInstance->getName()] ?? null,
-                    HandlerParameterAttributeInterface::UPLOADED_FILES => $request->getUploadedFiles()
-                };
+                $resolvedParameter = $resolver->resolve($attributeInstance, $request);
+                if ($resolvedParameter === null && $parameter->isDefaultValueAvailable()) {
+                    $resolvedParameter = $parameter->getDefaultValue();
+                }
+                $actionParameters[$parameter->getName()] = $resolvedParameter;
```
Comment (vjik) [inline src/HandlerParametersResolver.php:66]:
```suggestion
                if ($resolvedParameter !== null) {
                    $actionParameters[$parameter->getName()] = $resolvedParameter;
                }
```

File: README.md (hunk 1 above)
Comment (vjik) [reply README.md:35]: May be instead of this text add information about configuration to "Installation" section.

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by vjik — APPROVED** [review]

**Pull request merged**